src/pages/ContentScripts/features/repo-activity-openrank-trends.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render } from 'react-dom';
+import elementReady from 'element-ready';
+import $ from 'jquery';
+import * as pageDetect from 'github-url-detection';
+
+import features from '../../../feature-manager';
+import RepoActORTrendView from '../../../views/RepoActORTrendView/RepoActORTrendView';
+
+const featureId = features.getFeatureID(import.meta.url);
+
+async function init(): Promise<void> {
+  const repoName = pageDetect.utils.getRepositoryInfo(
+    window.location
+  )!.nameWithOwner;
+
+  const newBorderGridRow = document.createElement('div');
+  newBorderGridRow.id = featureId;
+  newBorderGridRow.className = 'BorderGrid-row';
+  const newBorderGridCell = document.createElement('div');
+  newBorderGridCell.className = 'BorderGrid-cell';
+  newBorderGridRow.appendChild(newBorderGridCell);
+
+  await elementReady('div.Layout-sidebar');
+  render(<RepoActORTrendView currentRepo={repoName} />, newBorderGridCell);
```

Comment:
But sometimes we do need to wait a certain element to show because we will inject a chart in that position. Don't worry, we have a way to wait for certain elements and that is `elementReady`.

Just call it with a selector in an async function, the execution will be paused until the selected element is ready.